atomica/migration.py

```diff
@@ -1,56 +1,172 @@
 # Central file for migrating Projects
-from distutils.version import LooseVersion
 from .system import logger, AtomicaException
 from .version import version
 import sciris as sc
 from .results import Result
 from .structure import FrameworkSettings as FS
+from distutils.version import LooseVersion
```

Comment:
Isn't this a standard library import? Shouldn't it stay above these others?

* https://pep8.org/#imports